src/benchmark/java/com/commercetools/sync/benchmark/BenchmarkUtils.java

```diff
@@ -44,10 +47,11 @@ final class BenchmarkUtils {
   static void saveNewResult(
       @Nonnull final String sync, @Nonnull final String benchmark, final double newResult)
       throws IOException {
-
-    final JsonNode rootNode = new ObjectMapper().readTree(getFileContent());
-    final JsonNode withNewResult = addNewResult(rootNode, sync, benchmark, newResult);
-    writeToFile(withNewResult.toString());
+    if (SUBMIT_BENCHMARK_RESULT) {
```

Comment:
More logical would be to put the if condition in the caller. When I call this method, I would expect it to save new results every time.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/868/commits/4426a2b4eaf895cbed477c288bcea1e884f55ed1